src/main/java/Car.java

```diff
@@ -0,0 +1,41 @@
+import javax.xml.stream.Location;
+
+public class Car {
+    int currentLocation;
+
+    Car() {
+        this.currentLocation = 0;
+    }
+
+    void moveForward() {
+        currentLocation++;
+    }
+
+    void play() {
+        final boolean madeFirstMove = makeFirstMove();
+        if (madeFirstMove) {
+            return;
+        }
+        makeMove();
+    }
+
+    private boolean makeFirstMove() {
+        if (isFirstMove()) {
+            moveForward();
+            return true;
+        }
+        return false;
+    }
+
+    private boolean isFirstMove() {
+        return currentLocation == 0;
+    }
+
+    private void makeMove() {
+        final int random = CarRacingRules.randomRange();
+        final boolean moveForward = CarRacingRules.determineToGoForwardOrStop(random);
```

Comment:
```suggestion
        final boolean moveForward = CarRacingRules.determineToGoForwardOrStop();
```

CarRacingRules에서 random 값을 꺼내와 다시 전달하는 방식이라면 위와 같이 구현해도 되지 않을까요?
테스트를 위함인가요?

Reply:
테스트를 위한 방법이긴 했는데 Car 와 CarRacingRules를 합치면서 다른 방식으로 변경했습니다.